corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/menu_list_spec.js

```diff
@@ -104,7 +102,7 @@ hqDefine("cloudcare/js/formplayer/spec/menu_list_spec", function () {
                 user.restoreAs = '';
                 user.displayOptions = {};
 
-                hqImport("cloudcare/js/formplayer/apps/api").primeApps(user.restoreAs, new Backbone.Collection());
+                hqImport("cloudcare/js/formplayer/apps/api").primeApps(user.restoreAs, []);
```

Comment:
This should pass a plain array, not a collection, because `getAppEntities` will wrap the given apps in a collection [here](https://github.com/dimagi/commcare-hq/blob/ffc6b81722bcf60ea15689f1bdf7cee2bba28f74/corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/api.js#L43-L46).

I think this passes because the test only deals with the specific test app it adds [here](https://github.com/dimagi/commcare-hq/blob/ffc6b81722bcf60ea15689f1bdf7cee2bba28f74/corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/menu_list_spec.js#L71-L73). The code that fails for me on a dev branch is a test that hits [this](https://github.com/dimagi/commcare-hq/blob/ffc6b81722bcf60ea15689f1bdf7cee2bba28f74/corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/controller.js#L13), because it iterates over all apps, but the apps array contains an item that's not an app, it's a collection.